expansion/src/test/java/no/unit/nva/publication/indexing/ExpandedResourceTest.java

```diff
@@ -145,10 +145,68 @@ void shouldReturnIndexDocumentWithCorrectTopLevelOrganization(int depth) throws
 
         assertThat(distinctTopLevelIds.stream().sorted().collect(Collectors.toList()),
                    is(equalTo(expectedTopLevelOrgs.stream().sorted().collect(Collectors.toList()))));
-
         assertExplicitFieldsFromFraming(framedResultNode);
     }
 
+//    @Test
+//    @Deprecated
+//    void shouldReturnIndexDocumentWithValidFundingSource() throws Exception {
+//
+//        final var publication = randomBookWithConfirmedPublisher();
+//        final var sourceUri0 = publication.getFundings().get(0).getSource();
+//        final var sourceUri1 = publication.getFundings().get(1).getSource();
+//        final var mockUriRetriever = mock(UriRetriever.class);
+//
+//        mockGetRawContentResponse(
+//            mockUriRetriever,
+//            sourceUri0,
+//            getPublicationSampleFundingSource(sourceUri0));
+//        mockGetRawContentResponse(
+//            mockUriRetriever,
+//            sourceUri1,
+//            getPublicationSampleFundingSource(sourceUri1));
+//
+//        final var framedResultNode = fromPublication(mockUriRetriever, publication).asJsonNode();
+//        final var extractedSourceId = extractSourceId(framedResultNode);
+//
+//        assertThat(extractedSourceId, hasItems(sourceUri0, sourceUri1));
+//    }
+
+    @Test
+    void shouldReturnIndexDocumentWithValidFundingSourceInsertingContextInFundingSource() throws Exception {
+
+        final var publication = randomBookWithConfirmedPublisher();
+        final var sourceUri0 = publication.getFundings().get(0).getSource();
+        final var sourceUri1 = publication.getFundings().get(1).getSource();
+        final var mockUriRetriever = mock(UriRetriever.class);
+
+        mockGetRawContentResponse(
+            mockUriRetriever,
+            sourceUri0,
+            getPublicationSampleFundingSourceWithoutContext(sourceUri0));
+        mockGetRawContentResponse(
+            mockUriRetriever,
+            sourceUri1,
+            getPublicationSampleFundingSourceWithoutContext(sourceUri1));
+
+        final var framedResultNode = fromPublication(mockUriRetriever, publication).asJsonNode();
+        final var extractedSourceId = extractSourceId(framedResultNode);
+
+        assertThat(extractedSourceId,hasItems(sourceUri0,sourceUri1));
```

Comment:
You could compare JsonNodes instead of extracted ids to assure that funding source is getting expanded as expected.